modules/beam/src/test/scala/com.snowplowanalytics.snowplow.enrich.beam/enrichments/IpLookupsEnrichmentSpec.scala

```diff
@@ -36,12 +36,12 @@ object IpLookupsEnrichmentSpec {
     "37.228.225.32",
     "10d96bc7-e400-4b29-8a41-6911ad00ee98",
     "IE",
-    "LH",
-    "Dundalk",
-    "A91",
-    "53.999",
-    "-6.4183",
-    "Louth",
+    "L",
+    "Dublin",
+    "D02",
+    "53.3331",
+    "-6.2489",
+    "Leinster",
```

Comment:
We should really take care of https://github.com/snowplow/enrich/issues/269 at some point